src/documents/css/idg.css.styl

```diff
@@ -178,16 +178,23 @@ a:hover .idg-logo {
     height: auto;
 }
 
-.idg-iconPrinciples svg,
+.idg-iconInsights svg,
 .idg-iconPractices svg,
 .idg-iconTools svg,
-.idg-iconActivities svg {
+.idg-iconActivities svg,
+.idg-iconDimensions svg {
     width: 1rem;
     height: 1rem; //width and height required by IE11.
 }
 
-.idg-iconPrinciples svg {
-    fill: principlesColor;
+.idg-iconDimensions svg {
```

Comment:
Since this is a style used only on the index, the classname should probably mention "index" in it so it's not confused with the general style .idg-iconDimensions.